src/Service.Host/client/src/containers/importBooks/tabs/ImpBookTab.js

```diff
@@ -56,37 +52,33 @@ const mapStateToProps = state => ({
         displayText: `${e.portCode} (${e.description})`,
         id: e.portCode
     })),
-    countries: countriesSelectors.getItems(state),
     currencies: currenciesSelectors.getItems(state)?.map(c => ({
         displayText: `${c.code} - ${c.name}`,
         id: c.code
-    }))
+    })),
+    parcelsSearchResults: parcelsSelectors.getSearchItems(state).map?.(p => ({
+        ...p,
+        id: p.parcelNumber,
+        name: p.parcelNumber.toString()
+    })),
+    parcelsSearchLoading: parcelsSelectors.getSearchLoading(state)
 });
 
 const initialise = props => dispatch => {
     if (
-        !props.employees ||
-        props.employees.length === 0 ||
-        !props.allSuppliers ||
-        props.allSuppliers.length === 0 ||
-        !props.transportCodes ||
+       !props.transportCodes ||
         props.transportCodes.length === 0 ||
```

Comment:
these OR's checking whether arrays are null or empty could be rewritten as a single NOT I think, e.g.
<code>
!props.transportCodes?.length
</code>